src/Dialog/Dialog.js

```diff
@@ -237,3 +265,5 @@ export default class Dialog extends Component {
     );
   }
 }
+
+export default withWidth()(Dialog);
```

Comment:
What about using a decorator instead? Something that we could be using to switch properties off and on based on the breakpoint? I see many use cases for that new Higher-order Component.